### ISSUE TITLE
Tutorials submenu order reversed

### ISSUE THREAD

**Issue by ghost** [rdwatters/hugo-docs-concept#11]
The order of the tutorials submenu is reversed. I took a look at the `site-navigation.html` partial, but I can't figure out why Tutorials are behaving differently.

**Comment by rdwatters** [comment]
I just set it to sort by weight, so I think we're good now. As an aside, I wonder whether any of these tutorials are still relevant. To see what I mean, compare the current docs site with this prototype. I've successfully incorporated the more important pieces into their respective sections...

Tutorials are tough if you don't set metrics for what shouldn't/shouldn't be included, so this isn't a knock on what's currently out there :smile:

**Comment by ghost** [comment]
Ah, I see the `.Site.Data.sitenavigation` part now. I didn't even think to look in the data folder.

Re: tutorials

I like what you've done with re-categorizing the tutorials. It makes much more sense that way. My first reaction was that there should still be a tutorial section, but after seeing how well most of the tutorials fit into already existing sections, I'm not so sure anymore.

- The multilingual tutorial is out of date and not needed anymore since `Hugo 0.17`. The existing content on Multilingual Mode is probably sufficient.
- Creating a Theme tutorial will be irrelevant once `https://hugodocs.info/themes/creating/` is finished
- The migration tutorial might be worth keeping, but can it fit in the Developer Tools section or create a Migrations section? If not, I guess it makes sense to keep the Tutorial section
- What about including Sam Osborne's [post on a calendar](https://discuss.gohugo.io/t/calendar-view/5186/7?u=sethm)? Or rwi's tutorial in the post after Sam's?

This might be something to ask people on the forum -- what would people like tutorials for?

Also, I was thinking that since this site uses so much of Hugo's functionality, it might be good to use some of this site's code for example code. For example, the [current example for Data Templates](https://hugodocs.info/templates/data-templates/#example-jaco-pastorius-solo-discography) is nice and might be worth keeping, but showing how you used the `site-navigation.html` partial to access data from `sitenavigation.yml` might be more useful for people examine. Or even the `homepagefeatures.toml` and `homepagetweets.toml`. I think those are far more relevant and better for newbies to examine.

I'm sure there are plenty of such cases. If you think that might be worth looking into, I can go through the existing examples and see which ones could be updated or added to in order to be more useful for people -- especially newbies.

**Comment by digitalcraftsman** [comment]
> Ah, I see the .Site.Data.sitenavigation part now. I didn't even think to look in the data folder.

spf13/hugo#3090 would support this as a feature.

> The multilingual tutorial is out of date and not needed anymore since Hugo 0.17. The existing content on Multilingual Mode is probably sufficient.

I agree. The built-in i18n feature is the way to go. However, I would add the section about the internationalization of dates as an example to the data file site. Currently, this workaround is the only possibiliy to achieve that.

**Comment by rdwatters** [comment]
> spf13/hugo#3090 would support this as a feature.

This is why I was bummed when Bjorn locked the "authors" conversation to just collaborators. My immediate thought was that the configs are too big, but it looks like the team is already on this. 

W/r/t tutorials, I agree with you guys. Future submission guidelines would be simple: a) don't elaborate on Hugo concepts already explained elsewhere in the docs (this creates a management nightmare) and b) the focus of the tutorial should be Hugo-specific rather than multiple 3rd-party services or libraries. So, for example, no more tutorials where half the content is on git (out of scope w/the exception of the contrib walkthrough) or  deployment (already described in detail in other areas of the site).

**Comment by rdwatters** [comment]
Note to self: closed for multiple reasons, not the least of which is the complete removal of the `tutorials` section from the site...
